spec/create_table_statement_spec.cr

```diff
@@ -73,4 +73,30 @@ describe LuckyMigrator::CreateTableStatement do
       end
     end
   end
+
+  describe "associations" do
+    it "can create associations" do
+      built = LuckyMigrator::CreateTableStatement.new(:comments).build do
+        belongs_to User
+        belongs_to Post?
+        belongs_to Category, references: :custom_table
```

Comment:
Could you make this something like `CategoryLabel` so that it tests that `underscore` is working correctly?

Reply:
Good idea, I changed it